src/content/1.7/modules/concepts/composer.md

```diff
@@ -111,7 +111,8 @@ correctly you **must not** override its dependencies. Which is why you need to a
 
 **Don't forget to publish your vendor folder**
 
-When using composer the autoload file generated by composer is required for your module to work correctly. So before
-creating your module archive and releasing it don't forget to run `composer dump-autoload -o` (or `composer install --no-dev` if
-you have included production dependencies) so that they will be included in your module.
+Composer is responsible in downloading your dependencies and generating the autoloader from the `composer.json`
+& `composer.lock` in your source code. These two artifacts are needed by your module to work properly, so before
+releasing it don't forget to run `composer dump-autoload -o --no-dev` to include your **production dependencies**
```

Comment:
```suggestion
releasing it don't forget to run `composer dump-autoload -o --no-dev` to only include your **production dependencies**
```

Reply:
There is something weird about this `only`. I wrote it at the beginning but decided to remove it, it's like we add the prod dependencies but we forget all the rest

Reply:
```suggestion
releasing it don't forget to run `composer dump-autoload -o --no-dev` to install your dependencies for a **production** environment
```
Maybe better? wdyt?